packages/ts/src/core/container/index.ts

```diff
@@ -61,9 +59,13 @@ export class ContainerCore {
       this.svg.append('defs').attr('class', 'svgDefs').html(config.svgDefs)
     }
 
-    // Apply the `aria-label` attribute
-    select(this._container)
-      .attr('aria-label', config.ariaLabel)
+    // use setTimout to allow _render to complete in the derived class
+    setTimeout(() => {
+      // Apply the `aria-label` attribute
+      select(this._container)
+        .attr('aria-label', this.getAriaDescription())
+        .attr('tabindex', '0')
```

Comment:
note this

Reply:
@ivegotwings Can you please explain in more detail whey do we need to delay rendering here?

Reply:
Also let's handle the logic related `ariaAutoLabel` and `ariaLabel` configuration here, not in the `getAriaDescription` method. See my comment below.

Reply:
We need this to complete the execution of `_render` function in the xy-container so as to generate the value of `this.xyDomain`which is later used in `getAriaDescription` 

If we do not use the delay `getAriaDescription` gets called before `the _render` method completes and the `this.xyDomain` remains undefined